README.md

```diff
@@ -306,14 +306,21 @@ For full example on Credit Card 3DS transaction refer to:
 ### 2.2.D Subscription API
 You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
 
+Subscription API only support for `Credit Card` and `Gopay`
+
+#### Subscription API for Credit Card
+
+To use subscription API for credit card, you should first obtain the 1 click token, [refer to docs](https://docs.midtrans.com/en/core-api/advanced-features?id=recurring-transaction-with-subscriptions-api)
+You will receive saved_token_id after complete 3ds authentication (it also available in the JSON of HTTP notification) [refer to docs](https://docs.midtrans.com/en/core-api/advanced-features?id=sample-3ds-authenticate-json-response-for-the-first-transaction)
```

Comment:
Check spacing on the rendered markdown, there's no new line. So please add new line and add proper dot (.) as sentence punctuation.

<img width="1049" alt="Screen Shot 2021-11-10 at 12 23 34" src="https://user-images.githubusercontent.com/13027142/141055789-9c9d9ba0-187c-46fe-ad01-47aa6be9a55e.png">

Reply:
Also grammar and context were incorrect, fix it to something like:
> To use subscription API for credit card, you should first obtain the 1-click saved token, [refer to this docs](https://docs.midtrans.com/en/core-api/advanced-features?id=recurring-transaction-with-subscriptions-api).
> You will receive `saved_token_id` as part of the response when the initial card payment is accepted (will also available in the HTTP notification's JSON), [refer to this docs](https://docs.midtrans.com/en/core-api/advanced-features?id=sample-3ds-authenticate-json-response-for-the-first-transaction).